clarity/tests/fixed-weight-pool_test.ts

```diff
@@ -32,145 +39,66 @@ Clarinet.test({
+        position = result.expectOk().expectTuple();
+        position['dx'].expectUint(ONE_8);
+        position['dy'].expectUint(4950465000000);    
         
-        // Create Pool
-        let result = FWPTest.createPool(deployer, gAlexTokenAddress, usdaTokenAddress, testWeightX, testWeightY, gAlexUsdaPoolAddress, 50000000000, 10000000000);
-        result.expectOk().expectBool(true);
-
-        // Create Pool Error : pool-already-exists
-        result = FWPTest.createPool(deployer, gAlexTokenAddress, usdaTokenAddress, testWeightX, testWeightY, gAlexUsdaPoolAddress, 50000000000, 10000000000);
-        result.expectErr().expectUint(2000);
-
-        // Add to Position : Invalid Pool Access
-        result = FWPTest.addToPosition(deployer, ayusdaAddress, usdaTokenAddress, testWeightX, testWeightY, gAlexUsdaPoolAddress, 500000000000, 100000000000);
-        result.expectErr().expectUint(2001);
-
-        // Add to Liquidity
-        result = FWPTest.addToPosition(deployer, gAlexTokenAddress, usdaTokenAddress, testWeightX, testWeightY, gAlexUsdaPoolAddress, 500000000000, 100000000000);
-        result.expectOk().expectBool(true);
-
-        // Invalid Liquidity - Edge case
-        result = FWPTest.addToPosition(deployer, gAlexTokenAddress, usdaTokenAddress, testWeightX, testWeightY, gAlexUsdaPoolAddress, 0, 0);
-        result.expectErr().expectUint(2003);
-
-        // Invalid Liquidity for checking assertion of dy, doesnt matter because it passes eqation and outputs new-dy.
-        result = FWPTest.addToPosition(deployer, gAlexTokenAddress, usdaTokenAddress, testWeightX, testWeightY, gAlexUsdaPoolAddress, 1000000, 0);
-        result.expectOk();
-
-        // Transfer Error due to lack of balance on sender
-        result = FWPTest.addToPosition(wallet_2, gAlexTokenAddress, usdaTokenAddress, testWeightX, testWeightY, gAlexUsdaPoolAddress, 500000000000, 100000000000);
-        result.expectErr().expectUint(3001);
-
-        // Math calling error 
-        // Used to be math calling error but on the update version of clarinet, it seems like it can find the error of math call, which is percent greater than one.
-        result = FWPTest.reducePosition(deployer, gAlexTokenAddress, usdaTokenAddress, testWeightX, testWeightY, gAlexUsdaPoolAddress, 1000000000000);
-        result.expectErr().expectUint(5000);
-
-        result = FWPTest.reducePosition(deployer, gAlexTokenAddress, usdaTokenAddress, testWeightX, testWeightY, gAlexUsdaPoolAddress, 99999);
-        result.expectOk();
-
-        // Internal Function call error due to max ratio error in weighted equation.
-        result = FWPTest.swapXForY(deployer, gAlexTokenAddress, usdaTokenAddress, testWeightX, testWeightY, 2000000000000);
-        result.expectErr().expectUint(1001);
-
-        // result = FWPTest.swapYForX(deployer, gAlexTokenAddress, usdaTokenAddress, testWeightX, testWeightY, 0);
-        // result.expectOk().expectUint(2003);
-
+        // swap some usda into wbtc
```

Comment:
Is it better to have swapping together in one test case then having independent one?